lookout/style/format/analyzer.py

```diff
@@ -62,10 +72,7 @@ def analyze(self, ptr_from: ReferencePointer, ptr_to: ReferencePointer,
                     lines = None
                 else:
                     lines = [find_new_lines(prev_file, file)]
-                X, y, vnodes = FeatureExtractor(language=lang,
-                                                siblings_window=self.config["siblings_window"],
-                                                parents_depth=self.config["parents_depth"],
-                                                debug_parsing=self.config["debug_parsing"]) \
+                X, y, vnodes = FeatureExtractor(language=lang, **config["feature_extractor"]) \
```

Comment:
Since we're in the analyze method, the parameters for feature extraction should come from the model, not from the config.